pkg/distro/generic/images.go

```diff
@@ -121,9 +114,9 @@ func osCustomizations(t *imageType, osPackageSet rpmmd.PackageSet, options distr
 
 	language, keyboard := c.GetPrimaryLocale()
 	if language != nil {
-		osc.Language = *language
+		osc.Language = common.ValueOrEmpty(language)
```

Comment:
Slightly curious, this is of course fine but given that in the line above we did the `if language != nil` check, why not keep it as is with the "naked" dereference? Its fine if you prefer it for e.e. consistency, mostly curious